pcsx2/gui/FrameForGS.cpp

```diff
@@ -173,6 +184,8 @@ void GSPanel::DoResize()
 		zoom = std::max( (float)arr, (float)(1.0/arr) );
 
 	viewport.Scale(zoom, zoom*g_Conf->GSWindow.StretchY.ToFloat()/100.0 );
+	if (viewport.x == (double)client.GetWidth() && viewport.y == (double)client.GetHeight() && EmuConfig.Gamefixes.FMVinSoftwareHack && g_Conf->GSWindow.IsFullscreen)
```

Comment:
Viewport values can get updated at lines 178, 180 and 186, the if statement checks if it's still the same as client after those possible updates. If it is, it increases the size of the viewport to not exactly equal client size, thus preventing the crash.